src/tokenserver/handlers.rs

```diff
@@ -50,7 +50,15 @@ pub async fn get_tokenserver_result(
         node_type: req.node_type,
     };
 
-    Ok(HttpResponse::build(StatusCode::OK).json(result))
+    let timestamp = {
+        let start = SystemTime::now();
+        start.duration_since(UNIX_EPOCH).unwrap().as_secs()
+    };
+    Ok(HttpResponse::build(StatusCode::OK)
+        .header("X-Content-Type-Options", "nosniff")
+        .header("X-Timestamp", timestamp.to_string())
+        .force_close()
```

Comment:
`syncstorage-rs` seems to optionally support keepalive based on the settings, so I'm not sure what the prod setting is. For now, Tokenserver will run as a separate server, and I've disabled keepalive for that server, but when we someday run Tokenserver as part of syncstorage, we may need to revisit whether we can keep setting `Connection: close` on Tokenserver responses if the server itself actually does support keepalive.

Reply:
I'm seeing `Connection: keep-alive` from [tokenserver prod](https://token.services.mozilla.com/)

Reply:
Oh I didn't think to check the `/` endpoint on prod...I'll remove the `.force_close()` 👍🏻